src/repo/repository.rs

```diff
@@ -549,7 +550,7 @@ impl Repository {
                 let object_readers = objects.iter().map(|&handle| {
                     // TODO: Method of obtaining readers from packs? Or we can
                     // just assume packs first get unpacked.
-                    Ok(Box::new(File::open(
+                    Ok(Box::new(open_file(
                         self.loose_object_path(index.handle_to_checksum(handle)),
```

Comment:
This fails to compile. Appears to be because `File::open` accepts anything that you can coerce to a `&Path`, while `open_file` strictly accepts `&Path`.